configuration/zebrunner-proxy/nginx.conf

```diff
@@ -0,0 +1,101 @@
+#IMPORTANT: don't use underscore in upstream server name otherwise Tomcat app server complain:
+#  The host [reporting-service_server] is not valid
+#  Note: further occurrences of request parsing errors will be logged at DEBUG level.
+#  java.lang.IllegalArgumentException: The character [_] is never valid in a domain name.
+
+user  nginx;
+worker_processes  1;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+
+    upstream reporting-ui-server {
+        server reporting-ui:80;
+    }
+
+    upstream reporting-service-server {
+        server reporting-service:8080;
+    }
+
+    upstream elasticsearch-server {
+        server elasticsearch:9200;
+    }
+
+    upstream rabbitmq-server {
+        server rabbitmq:15672;
+    }
+
+    server {
+        listen 80;
+
+        location /app {
+            proxy_pass http://reporting-ui-server/app/;
+        }
+
+        location /reporting-service/ {
```

Comment:
Minor note - would be nice to collapse two locations into one - have you finally figured out the right way to perform a logical OR? :) We should probably stick with `(reporing-service|zafira-ws)` instead of duplicating the same configuration snippet

Reply:
nope, still have in TODO list